go/.gitignore

```diff
@@ -0,0 +1 @@
+mcap/mcap
```

Comment:
can we put all binaries in a `bin` dir or something?

Reply:
I am +1 but think we can handle it in a separate patch - I know you have a ticket for it already. I'll merge this one.